media-tv/mythtv/mythtv-29.1-r3.ebuild

```diff
@@ -0,0 +1,373 @@
+	dev-qt/qtnetwork:5
+	dev-qt/qtscript:5
+	dev-qt/qtsql:5[mysql]
+	dev-qt/qtopengl:5
+	dev-qt/qtwebkit:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	media-gfx/exiv2:=
+	media-libs/freetype:2
+	media-libs/taglib
+	>=media-sound/lame-3.93.1
+	sys-libs/zlib
+	virtual/opengl
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXinerama
+	x11-libs/libXv
+	x11-libs/libXrandr
+	x11-libs/libXxf86vm
+	x11-misc/wmctrl
+	alsa? ( >=media-libs/alsa-lib-1.0.24 )
+	bluray? (
+		dev-libs/libcdio:=
+		media-libs/libbluray:=
+		sys-fs/udisks:2
+	)
+	cec? ( dev-libs/libcec )
+	dvd? (
+		dev-libs/libcdio:=
+		sys-fs/udisks:2
+	)
+	egl? ( media-libs/mesa[egl] )
+	fftw? ( sci-libs/fftw:3.0= )
+	hls? (
+		<media-libs/libvpx-1.8.0:=
+		>=media-libs/x264-0.0.20111220:=
+	)
+	ieee1394? (
+		>=media-libs/libiec61883-1.0.0
+		>=sys-libs/libavc1394-0.5.3
+		>=sys-libs/libraw1394-1.2.0
+	)
+	jack? ( media-sound/jack-audio-connection-kit )
+	lcd? ( app-misc/lcdproc )
+	libass? ( >=media-libs/libass-0.9.11:= )
+	lirc? ( app-misc/lirc )
+	perl? (
+		>=dev-perl/libwww-perl-5
+		dev-perl/DBD-mysql
+		dev-perl/HTTP-Message
+		dev-perl/IO-Socket-INET6
+		dev-perl/LWP-Protocol-https
+		dev-perl/Net-UPnP
+	)
+	pulseaudio? ( media-sound/pulseaudio )
+	python? (
+		${PYTHON_DEPS}
+		dev-python/lxml
+		dev-python/mysql-python
+		dev-python/urlgrabber
+		dev-python/future
+		dev-python/requests-cache
+	)
+	systemd? ( sys-apps/systemd:= )
+	theora? ( media-libs/libtheora media-libs/libogg )
+	vaapi? ( x11-libs/libva:=[opengl] )
+	vdpau? ( x11-libs/libvdpau )
+	vorbis? ( >=media-libs/libvorbis-1.0 media-libs/libogg )
+	xml? ( >=dev-libs/libxml2-2.6.0 )
+	xvid? ( >=media-libs/xvid-1.1.0 )
+	zeroconf? (
+		dev-libs/openssl:0=
+		net-dns/avahi[mdnsresponder-compat]
+	)
+"
+RDEPEND="${COMMON}
+	media-fonts/corefonts
+	media-fonts/dejavu
+	media-fonts/liberation-fonts
+	x11-apps/xinit
+	autostart? (
+		net-dialup/mingetty
+		x11-apps/xset
+		x11-wm/evilwm
+	)
+	dvd? ( media-libs/libdvdcss )
+	xmltv? ( >=media-tv/xmltv-0.5.43 )
+"
+DEPEND="${COMMON}
+	dev-lang/yasm
+	virtual/pkgconfig
+	x11-base/xorg-proto
+"
+
+S="${WORKDIR}/${P}-r1/mythtv"
+
+DISABLE_AUTOFORMATTING="yes"
+DOC_CONTENTS="
+If a MYSQL server is installed, a mythtv MySQL user and mythconverg database
+is created if it does not already exist.
+You will be prompted for your MySQL root password.
+
+To have this machine operate as recording host for MythTV,
+mythbackend must be running. Run the following:
+rc-update add mythbackend default
+
+Your recordings folder must be owned 'mythtv'. e.g.
+chown -R mythtv /var/lib/mythtv
+
+Want mythfrontend to start automatically?
+Set USE=autostart. Details can be found at:
+https://dev.gentoo.org/~cardoe/mythtv/autostart.html
+
+Note that the systemd unit now restarts by default and logs
+to journald via the console at the notice verbosity.
+"
+
+MYTHTV_GROUPS="video,audio,tty,uucp"
+
+PATCHES=(
+	"${FILESDIR}/${P}-exiv2-0.27.1.patch"
+	"${FILESDIR}/${PN}-Fix_create_webbrowser_window.patch"
```

Comment:
patch should be prefixed with `${P}` (first version this applies to) instead. bonus points for adding bug ref.

Reply:
This same patch applies to all versions. It was back ported upstream.

Reply:
as I said.

Reply:
I made the change you requested for mythtv-29.1-r3.ebuild and changed mythtv-30.0_p20190808-r3.ebuild to use the same patch file.